wix-bookings/guides/Terminology.md

```diff
@@ -1,58 +1,57 @@
-| Course schedule | For classes and courses, the business decides when to schedule a session. This is in contrast to appointments, where the customer decides when to book a session. For classes and courses, the schedule is used to manage the available sessions and holds some aggregated information, such as the start time and end time of a course. | 
+| [Appointment](https://support.wix.com/en/article/creating-the-right-booking-service-for-your-business#appointments) | A specific time slot that a customer can book in a calendar. For example, a hair salon is open Monday to Friday from 9:00 AM to 7:00 PM. Customers can book a session at any available time during these hours. |
+| Appointment schedule | Defines when customers can book a session for an appointment service. Bookings are based on the business’s opening hours, session duration, time between sessions, and staff availability. For example, a hair salon operates Monday to Friday, 9:00 AM to 7:00 PM, offering 30-minute and 1-hour haircuts. One staff member is available from 9:00 AM to 5:00 PM, and another from 11:00 AM to 7:00 PM, with 15-minute buffers between sessions. Customers can book an appointment if the service and at least one staff member are available, considering the 15-minute buffer before and after the session. |
+| Availability | Free time on an appointment schedule that customers can book. |
+| Booking | Information about the service that the customer has booked. You can think of a booking as an order placed by a customer for a service. Includes details about the time, location, participants, and price. |
+| Business owner | The owner of the business that offers services through Wix. Can be a different person than the site owner. The business owner makes decisions such as confirming or declining a booking.
+| [Calendar](https://support.wix.com/en/article/wix-bookings-about-the-wix-booking-calendar) | General overview about the availability and bookings of the business, including its services and resources. You can manage appointments, classes, and courses on the calendar. |
+| `CANCELED` (Booking status) | Indicates that the booking has been canceled by the site owner or the customer. Cancel bookings using [Cancel Booking](https://dev.wix.com/docs/rest/business-solutions/bookings/bookings-and-time-slots/bookings-v2/bookings-v2-and-confirmation/bookings-cancel-booking). |
+| Catalog | A list of all the different services a business provides. |
+| Category (Service)| A label used to group services, helping business owners, your app, and customers to filter and organize services efficiently. |
+| Checkout | Customers must complete the [checkout](https://dev.wix.com/docs/rest/business-solutions/e-commerce/checkout/introduction) when booking a service. |
```

Comment:
we should add a sentence that defines what the checkout is, instead of defining a term with the same term.

Reply:
Thanks, I've copied a sentence from eCom.